CHANGELOG.md

```diff
@@ -1,62 +1,47 @@
 # Changelog
 
-### [1.0.1](https://www.github.com/googleapis/ruby-spanner-activerecord/compare/activerecord-spanner-adapter/v1.0.0...activerecord-spanner-adapter/v1.0.1) (2022-04-20)
+### 1.0.0 (2021-12-07)
```

Comment:
It seems that we lose all links in this file. Is this expected?

Reply:
I intentionally removed them because I don't consider them useful. Our other Ruby libraries (e.g. client libraries) don't include them, and the Ruby changelog template for release-please 13 has them removed so they won't appear for future releases. Does the spanner team want to keep them?

Reply:
Okay, it's good to be consistent with others.